uint8 masterID
string stateReq
string subState
---
int16 stateRes
